synonymdev/bitkit-react-native
# test: e2e lightning

### Description

e2e test for lightning

Test plan:
- connect to LND node
- receive funds
- send funds
- check balances, tx history and notes
- restore wallet
- check balances, tx history and notes
- close channel


### Type of change

Refactoring

### Tests

Detox test

### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/155891/505cf3ca-5327-4beb-8ad8-3dc40d0d89ba
